compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -155,7 +156,13 @@ void set_minmax_to_non_const(loco::Graph *g, float min, float max)
   }
 }
 
-class LogisticTestGraph final : public luci::test::TestIOGraph
+class SingleOpTestGraph : public luci::test::TestIOGraph
```

Comment:
Why did you introduce `SingleOpTestGraph` ?

Reply:
@seanshpark Because the LogisticTestGraph and others inherits TestIOGraph but not init(Shape, Shape) function. Instead, they implement init(void) function, which doesn't exists in TestIOGraph.  SingleOpTestGraph has init(void) that initializes its nodes with its specific shapes.

Reply:
> SingleOpTestGraph has init(void) that initializes its nodes with its specific shapes.

Can you explain more about this? what do you mean by `specific shapes.` ?

Reply:
@seanshpark These tests are setting input/output node's shape simply as {32} in init(void) function.
The TestIOGraph class's init(shape, shape) function asks user about what shape to use for input/output.
I thought that Single model graphs in QuantizedModelVerifier.test.cpp are not interested in testing all the shapes, so they need the init(void) function, which just initializes io nodes with pre-set shapes.

Reply:
So you mean, you want to provide a fixed/predefined input that has a single Op? I assume single op here is not that important (you I maybe wrong)
What makes me confusing: 
- By the name `SingleOpTestGraph`: I expect that this class will provide some restriction to `TestIOGraph` so that that there will be a ONLY single op as the class name explains.

I think it would be better to change the name of the class to reflect what you want to provide.

Reply:
@seanshpark Sure, I will think about that :-D

Reply:
I changed the name into SampleTestGraph as they are all being initialized with specific shapes and values selected by naive sampling.

Reply:
`SampleTestGraph` looks OK,
how about `SimpleTestGraph` or `SimpleInputTestGraph` ?

Reply:
@seanshpark Sure. :-D SimpleTestGraph looks fine.